### ISSUE TITLE
high volume of users error

### ISSUE THREAD

**Issue by umutxmen** [YarmolenkoD/moonbix#36]
![c8ac7f4f-49fc-413e-819b-5e25ac6b7c6b](https://github.com/user-attachments/assets/a6a28f10-b28e-4660-a9d4-e3e788a3df23)
I am using the bot, but I think there is such a problem due to the API. How can we use the faker plugin or something?

**Comment by YarmolenkoD** [comment]
@umutxmen I think it's an internal problem they've written to all people about, whether they use our script or not.

**Comment by umutxmen** [comment]
> Bence bu, senaryomuzu kullansalar da kullanmasalar da tüm insanlara yazdıkları bir iç sorun.

It only happens on accounts where I use the bot. And even if I close the bot, it doesn't work for hours.

**Comment by XPacL** [comment]
> > Bence bu, senaryomuzu kullansalar da kullanmasalar da tüm insanlara yazdıkları bir iç sorun.
> 
> It only happens on accounts where I use the bot. And even if I close the bot, it doesn't fix for hours.

Yeah, on all my accounts on which the script is running have this message. My other accounts on which I did not run the script do not have such a message.
